conftest.py

```diff
@@ -24,6 +24,7 @@
 import numpy as np
 import pytest
 import requests
+import tensorflow as tf
```

Comment:
Please move import closer to application, the top-level import would create dependency.